.devcontainer/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.191.1/containers/ruby/.devcontainer/base.Dockerfile
+
+# [Choice] Ruby version: 3, 3.0, 2, 2.7, 2.6
+ARG VARIANT="3.0"
+FROM mcr.microsoft.com/vscode/devcontainers/ruby:0-${VARIANT}
+
+# [Choice] Node.js version: none, lts/*, 16, 14, 12, 10
+ARG NODE_VERSION="none"
+RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+
+# [Optional] Uncomment this section to install additional OS packages.
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends \
+        mariadb-server libmariadbclient-dev \
```

Comment:
My mariadb and not mysql?

Reply:
I'm more familiar with PostgreSQL, so I needed to look this up when I was writing the Dockerfile. Since [Debian 9](https://mariadb.com/kb/en/moving-from-mysql-to-mariadb-in-debian-9/), MySQL is no longer available in the default apt repositories. MariaDB is the new default database engine. [Because MariaDB is backwards compatible with MySQL and in most situations a better choice for new projects](https://stackoverflow.com/a/16187934), I assumed it isn't a problem to follow the OS default database engine approach instead of doing more work to install MySQL.

Because Github actions still run MySQL, compatibility with a real MySQL installation would still be validated when making code changes.